prow/config/tide.go

```diff
@@ -224,6 +229,16 @@ func (t *Tide) GetPRStatusBaseURL(repo OrgRepo) string {
 	return t.PRStatusBaseURLs["*"]
 }
 
+func (t *Tide) GetTargetURL(repo OrgRepo) string {
+	if byOrgRepo, ok := t.TargetURLs[repo.String()]; ok {
+		return byOrgRepo
+	} else if byOrg, ok := t.TargetURLs[repo.Org]; ok {
```

Comment:
else is not needed since already returned

Reply:
Changed here and `GetPRStatusBaseURL`